src/client/components/componentWithSliders/savePressetModalBlock/modalSavePressetesWindow/modalSavePressetesWindow.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './modalSavePressetesWindow.css';
+import { cancelWindowIcon } from '../../../../assets/icons/icons';
+import SavePressetsToBDAndCancelButton from './savePressetsToBDAndCancelButton';
+
+const SavePressetesModalWindow = (props) => {
+  const { saveButtonClick, showHideModalBlock, refFocus } = props;
+  return (
+    <div
```

Comment:
I think it will be better to use 'button' (tag) in this case.

Reply:
it's modal window -  I use div for this. "role button" for div linter needed.